app/graphql/types/query_type.rb

```diff
@@ -3,7 +3,7 @@
 class QueryType < BaseObject
   extend_type
 
-  field :members, MemberConnectionType, null: false, connection: true, max_page_size: 1000 do
```

Comment:
In general it looks ok. However I do think it would help as a reviewer that there different commits for different things. I guess this is about pagination and it has nothing to do with the actor interface. its totally fine its in the same PR but maybe use more than one commit so its easier to read the code during the review